src/local_scheduler/local_scheduler_algorithm.cc

```diff
@@ -230,10 +237,11 @@ void create_actor(SchedulingAlgorithmState *algorithm_state,
                   LocalSchedulerClient *worker) {
   /* This will be freed when the actor is removed in remove_actor. */
   LocalActorInfo *entry = (LocalActorInfo *) malloc(sizeof(LocalActorInfo));
+  memset(entry, 0, sizeof(LocalActorInfo));
```

Comment:
The (semi-)C++ way to do this would be `LocalActorInfo *entry = new LocalActorInfo()`, and to then define `LocalActorInfo::LocalActorInfo() : member1(), member2(), member3(), ... { }`, and finally call `delete` after.

(The full C++ way would use something like `std::unique_ptr` etc. which I'd recommend gradually switching wherever you're finding yourself making other changes like these.)